components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway/src/main/resources/gateway-api.yaml

```diff
@@ -58,52 +58,25 @@ paths:
   /redeploy-api:
 
     #-----------------------------------------------------
-    # Deploy the API in the Gateway
+    # Redeploy the API in the Gateway
     #-----------------------------------------------------
     post:
-      x-wso2-curl: "curl -k -H \"Authorization: Basic YWRtaW46YWRtaW4= \" -X POST \"https://127.0.0.1:9443/api/am/gateway/v2/redeploy-api?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com\""
-      x-wso2-request: |
-        POST https://127.0.0.1:9443/api/am/gateway/v2/redeploy-api?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com
-        Authorization: Basic YWRtaW46YWRtaW4=
-      x-wso2-response: "HTTP/1.1 200 OK"
       summary: Re Deploy the API in the gateway
       description: |
         This operation is used to re deploy an API in the gateway. If the Tenant domain is not provided carbon.super
         will be picked as the Tenant domain.
+      operationId: redployAPI
       parameters:
-        - name: apiName
-          description: |
-            Name of the API
-          in: query
-          type: string
-          required: true
-        - name: version
-          description: |
-            version of the API
-          in: query
-          type: string
-          required: true
-        - name: tenantDomain
-          description: |
-            Tenant Domain of the API
-          in: query
-          type: string
-
-      tags:
-        - Deploy API
+        - $ref: '#/parameters/ApiName'
+        - $ref: '#/parameters/Version'
+        - $ref: '#/parameters/TenantDomain'
```

Comment:
```suggestion
        - $ref: '#/parameters/apiName'
        - $ref: '#/parameters/version'
        - $ref: '#/parameters/tenantDomain'
```

Reply:
shall we fix other places as well?

Reply:
Fixed with cc8ecdc730f39566a3072c901a1c2f992bfd09b2